src/briefcase/platforms/windows/__init__.py

```diff
@@ -88,6 +88,7 @@ def run_app(self, app: BaseConfig, **kwargs):
                 [os.fsdecode(self.binary_path(app))],
                 cwd=self.home_path,
                 check=True,
+                stream_output=True,
```

Comment:
While this doesn't actually get the output in to the log for Windows apps (see https://github.com/beeware/briefcase/pull/782#discussion_r920621894), the consistency seems appropriate.